modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/configuration/admin/display/FeatureFlagConfigurationScreen.java

```diff
@@ -97,6 +102,11 @@ public void render(
 		}
 	}
 
+	private static final List<FeatureFlagType> _featureFlagsConfigurationOrder =
+		Arrays.asList(
+			FeatureFlagType.RELEASE, FeatureFlagType.BETA,
+			FeatureFlagType.DEPRECATION, FeatureFlagType.DEV);
+
 	private final FeatureFlagManager _featureFlagManager;
```

Comment:
Do you mind removing the `_featureFlagManager` in a new "SF" commit? I just noticed it's not used anywhere in the class.

Reply:
Nice catch... removing it now.